src/resources/NotFoundSite/NotFoundSite.module.css

```
@@ -0,0 +1,47 @@
+.notFoundSiteWrapper {
+  margin-left: 20px;
```

This is very much nitpick, but...
```suggestion
  margin-left: 40px;
```